R/opart_gaussian.R

```diff
@@ -57,7 +57,7 @@ opart_gaussian <- function(data, penalty) {
 
   #remove the columns used for internal calculations as they don't need to be displayed
   #result <- result[-c(5,6,8)]
-  result <- result[!(names(result) %in% c("sums","dp","positions"))]
+  result <- result[!(names(result) %in% c("sums","sums_sq","dp","positions"))]
```

Comment:
@as4378 probably better to select the names of the list elements which you DO want to return